src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.Android/Avalonia.Labs.Notifications.Android.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net8.0-android</TargetFramework>
```

Comment:
Why won't we have it as a single multi-target project? Should be simple enough until each target won't have unnecessary dependencies.

Reply:
I'll update the projects.

Reply:
They are all merged now.